mitodl/micromasters
# Added ability to mark order as refunded

#### What are the relevant tickets?
Fixes #1420 

#### What's this PR do?
Adds a refunded state to the Order and provides a way to let the user change the order status in the Django Admin.

#### How should this be manually tested?
Change an `Order` status via the Django admin, and verify that the change is reflected in the `OrderAudit` table
